### PR TITLE
[KO] Update `OWC 2023` to RO32

### PR DESCRIPTION
**Pull request by Chronicler01**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Review by Sitoria — APPROVED** [review]
copypasta moment
(jk translate country names n notes)

File: wiki/Tournaments/OWC/2023/ko.md [hunk 1]
```diff
@@ -146,12 +198,78 @@ osu! 월드컵 2023은 [osu! 팀](/wiki/People/osu!_team)과 여러 명의 커
   2. [A-One feat. Shihori - Bamboo Dance (nanoya) \[Lunatic\]](https://osu.ppy.sh/beatmapsets/2075280#osu/4344475)
   3. [Sable Hills - Not Falling (squirrelpascals) \[Vid\]](https://osu.ppy.sh/beatmapsets/2075253#osu/4344422)
 
+## 결과
+
+### 퀄리파이어
+
+퀄리파이어 최종 순위는 [스프레드시트](https://docs.google.com/spreadsheets/d/1iih6qvAYpCakmdZcnFoB_XGxBm4WDBYNNeo-81PcKGc?rm=minimal)에서 확인할 수 있습니다.
+
+| 시드 | 국가 | %MAX sum[^qualifiers-seeding] | avg. score[^qualifiers-tiebreaker] |
```
Comment (cl8n) [inline wiki/Tournaments/OWC/2023/ko.md:207]: this doesn't seem to be the most updated version of the table from en

Comment (Chronicler01) [reply wiki/Tournaments/OWC/2023/ko.md:207]: if you mean https://github.com/ppy/osu-wiki/commit/ed45fad16e42ccd84333b37d728315d085a6c16a, i've done

Spreadsheet link is same

**Comment by Chronicler01** [comment]
all done

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**